src/core/event-core.js

```diff
@@ -1,8 +1,39 @@
 import _ from 'lodash';
+const BPromise = require('bluebird');
 const {knex} = require('../util/database').connect();
 import {getDistance} from '../util/geometry';
 import moment from 'moment-timezone';
+import { getFeed } from "./feed-core";
 
+/**
+ *
+ * @param {number} opts.eventId
+ * @param {number} opts.client.id
+ */
+function getEventById(opts = {}) {
+  return getEvents(opts)
```

Comment:
I thought about making a single query here that fetches everything, but making two queries shouldn't be that huge perf penalty and this way we can better follow DRY